index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+import { bookManager } from './modules/bookManager.js';
```

Comment:
- [ ] Please don't disable linter errors instead fix them manually, It will help you to write more clean and professional code.